### ISSUE TITLE
Output of catch-all routes broken since v4.1.13

### ISSUE THREAD

**Issue by codeflorist** [harlan-zw/nuxt-simple-sitemap#191]
### Describe the bug

Output of dynamic routes (via `[...slug].vue`) in the sitemap seems to be broken from v4.1.13 onwards.

It was working in v4.1.12 as shown in this stackblitz:
https://stackblitz.com/edit/nuxt-starter-wt1gzo?file=package.json

### Reproduction

https://stackblitz.com/edit/nuxt-starter-iuktax?file=package.json

### System / Nuxt Info

```Shell
------------------------------
- Operating System: Linux
- Node Version:     v18.18.0
- Nuxt Version:     3.8.2
- CLI Version:      3.10.0
- Nitro Version:    2.8.1
- Package Manager:  npm@9.4.2
- Builder:          -
- User Config:      devtools, modules, nitro, site, sitemap
- Runtime Modules:  nuxt-simple-sitemap@4.1.13
- Build Modules:    -
------------------------------
```

**Comment by harlan-zw** [comment]
Thank you for the reproduction, looks fixed in v4.1.17.

**Comment by codeflorist** [comment]
looks good! thanks for your impeccable support! 👍
